### DISCUSSION TITLE
posts/%E6%A3%80%E6%B5%8B%E5%9B%BE%E7%89%87%E6%98%AF%E5%90%A6cmyk/

### DISCUSSION THREAD

**Discussion by giscus** [joisun/blog_11ty#2] [General]
# posts/%E6%A3%80%E6%B5%8B%E5%9B%BE%E7%89%87%E6%98%AF%E5%90%A6cmyk/

simple style blog site

http://localhost:8080/posts/%E6%A3%80%E6%B5%8B%E5%9B%BE%E7%89%87%E6%98%AF%E5%90%A6cmyk/

**Comment by joisun** [comment]
hi how you doing
